api/things.yml

```diff
@@ -636,7 +636,10 @@ components:
         id:
           type: string
           format: uuid
-          description: Thing unique identifier
+          description: Thing unique identifier.  This can be either
```

Comment:
Looks like you are adding 2 white space after every dot.

Reply:
Family speaking it was not me, but fixed. ^_^